### DISCUSSION TITLE
Typo on the Website

### DISCUSSION THREAD

**Discussion by fichtelmann** [esnet/iperf#1486] [General]
I don't want to raise an issue about this. I've seen a small typo on the docu website: https://iperf.fr/iperf-doc.php

Regarding the option for IPv6 '-6' the long version written down there is '--version4' but it should be '--version6'

**Comment by WatcherOfTheSkies** [comment]
Hi,

I saw a discussion that states that website is a 'fan' site and not under the jurisdiction of the maintainers here. I believe they have tried to reach out to them too without success.

Regards.

WatcherOfTheSkies.

**Comment by bmah888** [comment]
@fichtelmann I see the problem, thanks for finding and pointing that out! As @WatcherOfTheSkies noted, we don't have any control over that site and all of our attempts to contact the maintainer or maintainers (over the past 5+ years) have failed.

The documentation site maintained by the iperf3 developers is: https://software.es.net/iperf/

Bruce.
